order-enhancer/pom.xml

```diff
@@ -41,7 +39,7 @@
     <dependency>
       <groupId>io.smallrye.reactive</groupId>
       <artifactId>smallrye-reactive-messaging-in-memory</artifactId>
-      <version>2.7.0</version>
+      <version>4.9.0</version>
```

Comment:
It should be in the Quarkus BOM now, can you check and remove the version if it's managed there?

Reply:
in deed, it is better ths way